fem/gridfunc.cpp

```diff
@@ -194,6 +201,26 @@ void GridFunction::Update()
    if (t_vec.Size() > 0) { SetTrueVector(); }
 }
 
+void GridFunction::UpdatePRef()
+{
+   const std::shared_ptr<const PRefinementTransferOperator> Tp =
+      fes->GetPrefUpdateOperator();
+   if (Tp)
+   {
+      Vector old_data;
+      old_data.Swap(*this);
+      MFEM_VERIFY(Tp->Width() == old_data.Size(),
```

Comment:
The `MFEM_VERIFY` fails for me here if I update a vector `gridfunction`.

Reply:
This issue resolves if we account for `vdim` and `ordering` when constructing `fesPrev`. See my other comments below.

Reply:
This works now with the bug fixes you suggested.